lib/core/include/irods/osauth.h

```diff
@@ -6,9 +6,7 @@
 #define OS_AUTH_H__
 
 
-#define OS_AUTH_CMD      "var/lib/irods/clients/bin/genOSAuth"
 #define OS_AUTH_ENV_USER "OSAUTH_USERNAME"
-#define OS_AUTH_KEYFILE "etc/irods/irods.key"
```

Comment:
Hmm, these appear to be part of the public API. We need to decide whether these macros can be moved. Based on the values, they look like they shouldn't be used on the client-side.

@trel @alanking
- What kind of support will the OS auth plugin receive in 4.3.0?
- Is it just not being ported to the new auth framework?
- Are we still supporting the legacy auth plugin?

Reply:
I think we are... not shipping osauth with 4.3.0.   If anyone wants it, we can discuss a plugin later - or just configure PAM to provide the same effect (require a local authenticated user... like our PAM training).   @alanking is that right?

Reply:
Ok. Since we aren't planning to carry OS auth forward into 4.3, these changes are fine.

Reply:
Yes, that's right. Should we need to ship it in the future (unlikely) we can always find a different way forward.